lib/matplotlib/colors.py

```diff
@@ -114,6 +114,11 @@ class ColorSequenceRegistry(Mapping):
     `.ColorSequenceRegistry.register`::
 
         mpl.color_sequences.register('rgb', ['r', 'g', 'b'])
+
+    To get a list of all registered colormaps, you can do::
+
+        from matplotlib import colormaps
+        list(colormaps)
```

Comment:
I'm not sure this is the right place; it appears to be in the colour sequences section.

Reply:
My mistake, should be fixed now!